### DISCUSSION TITLE
J2000.0 ecliptic and equator lines at very small FOV

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#3005] [General]
Stellarium can reach an incredible minute FOV of just 0.00139º. But at that extreme zoom even the theoretical ecliptic and equator lines at J2000.0 may not match. Some other planetarium programs do solve the issue by limiting much more the power to zoom in. Beyond a certain FOV, lines just vanish. I do not know how Stellarium could fix that, or even if it's worth the effort, since it may simply be a decimals limit calculation problem. Just a commentary.
![image](https://user-images.githubusercontent.com/59968775/213429659-4ddb11b2-ee21-4233-b8f7-772d317d03e0.png)

**Comment by 10110111** [comment]
What do you mean by "the theoretical ecliptic and equator lines at J2000.0 may not match"? Is there a very small difference by definition, that would become visible, or do you just mean that software might become inaccurate and show a difference while there should be none?

**Reply by diegorodriguezperegrin** [reply]
What is this "small difference by definition"?  The mean equinox of J2000.0 is shifted from the ICRS right ascension origin by 78±10 mas (or 0° 0' 0.078", if I am not wrong). From what I can measure in the printscreen, there must be around 0° 0' 0.07" difference, so it could well be it.

**Reply by 10110111** [reply]
OK, if the shift is expected to be there, why should Stellarium "fix" this?

**Reply by gzotti** [reply]
I cannot say whether we should call our J2000.0 grid "ICRS". There is a minuscule rotation between VSOP and J2000.0, maybe this is what we are seeing here? For all practical purposes we are happy with the current state of subarcsecond accuracy, despite possible final systematic offsets against IAU ICRS. Else please dig very deeply into the core of this project, and dig even deeper than I did.

**Comment by gzotti** [comment]
Probably you should try with nutation switched off. But yes, there is some offset of 0.1 arcsecond. 
You can limit min_fov. See User Guide, D.1.5.

**Comment by diegorodriguezperegrin** [comment]
By using your excellent tool to show coordinates of the mouse pointer, I saw that 0,0 comes on the Equator of J2000.0 line, So it's the ecliptic line not "behaving".

![image](https://user-images.githubusercontent.com/59968775/213527836-7abafd24-080d-4174-8967-4d76276c34ff.png)
